functional_tests/scylla_operator/libs/auxiliary.py

```diff
@@ -39,13 +41,20 @@ def update_test_status(self, test_name, status, error=None):
         self.test_data[test_name] = (status, error)
 
     def get_test_failures(self):
-        pass
+        for test_name, test_data in self.test_data.items():
+            if not test_data[0] in ('SUCCESS', 'SKIPPED'):
+                TestFrameworkEvent(
+                    source=self.__class__.__name__,
+                    source_method=test_name,
+                    message=test_data[1],
+                    severity=Severity.ERROR,
+                ).publish_or_dump()
 
     def get_test_status(self):
         for _, test_data in self.test_data.items():
             if not test_data[0] in ('SUCCESS', 'SKIPPED'):
                 return 'FAILED'
-        return 'SUCCESS'
+        return 'SUCCESS' if self.test_data else 'FAILED'
```

Comment:
As I mentioned in the draft PR of @k0machi , we should not report 'SUCCESS' when there were only 'SKIPPED' tests. It must be 'ABORTED' or 'SKIPPED' result.

Reply:
Why ?

I don't know of any test runner that apply this logic, they all report success on those cases.